### PR TITLE
Use the group id of the parent part when creating a new module

### PR DESCRIPTION
**Pull request by simong**
This fixes a bug where a new module is created under an empty part.

### CODE REVIEW

File: shared/gh/js/views/tenant-admin/gh.admin-batch-edit.js [hunk 1]
```diff
@@ -967,7 +967,10 @@ define(['gh.core', 'gh.constants', 'moment', 'moment-timezone', 'gh.calendar', '
         var createNewEvent = function(newEvent, _callback) {
             // Get the ID of the series this event is added to
             var seriesId = parseInt(History.getState().data['series'], 10);
-            gh.api.eventAPI.createEvent(newEvent.displayName, newEvent.start, newEvent.end, null, null, newEvent.location, newEvent.notes, newEvent.organiserOther, newEvent.organiserUsers, seriesId, newEvent.type, function(evErr, data) {
+            // Get the Group Id of the part under which this series is being added
+            var partId = parseInt(History.getState().data['part'], 10);
+            var part = gh.utils.getPartById(partId);
```
Comment (nicolaasmatthijs) [inline shared/gh/js/views/tenant-admin/gh.admin-batch-edit.js]: Do we need any handling here in case partId is invalid or part can't be found ?

**Comment by nicolaasmatthijs** [comment]
Left a minor comment. Back to @simong

File: shared/gh/js/views/tenant-admin/gh.admin-batch-edit.js (hunk 1 above)
Comment (simong) [inline shared/gh/js/views/tenant-admin/gh.admin-batch-edit.js]: Pushed some follow-up where we mark each new event row as red and show an error notification

File: shared/gh/js/utils/gh.utils.orgunits.js [hunk 2]
```diff
@@ -65,6 +65,16 @@ define(['exports'], function(exports) {
     };
 
     /**
+     * Get a part by its id
+     *
+     * @param  {Number}     partId      The id of the part to retrieve
+     * @return {Object}                 The organisational unit that represents the part
+     */
+    var getPartById = exports.getPartById = function(partId) {
```
Comment (Coenego) [inline shared/gh/js/utils/gh.utils.orgunits.js:73]: This will need a unit test to keep the coverage up to 100%

**Comment by Coenego** [comment]
Looks good. One comment left regarding a missing unit test.

**Comment by simong** [comment]
Done

**Pull request merged**